db/monitor/DELD06F.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<monitor name="Dell U2515H" init="standard">
+	<include file="DELD070.xml"/>
```

Comment:
TravisCI screems with error:

```
I/O warning : failed to load external entity "./monitor/DELD070.xml.xml"
```

No need to add `.xml`, it's appended automatically.